packages/react-instantsearch-hooks-server/package.json

```diff
@@ -30,7 +37,8 @@
     "clean": "rm -rf dist",
     "watch": "yarn build:cjs --watch",
     "build": "yarn build:cjs && yarn build:es && yarn build:umd && yarn build:types",
-    "build:cjs": "babel src --root-mode upward --extensions '.js,.ts,.tsx' --out-dir dist/cjs --ignore '**/__tests__/**/*','**/__mocks__/**/*' --quiet",
+    "build:cjs": "babel src --root-mode upward --extensions '.js,.ts,.tsx' --out-dir dist/cjs --ignore '**/__tests__/**/*','**/__mocks__/**/*' --quiet && yarn build:cjs-replace-imports",
+    "build:cjs-replace-imports": "find dist/cjs -type f -exec sed -i.bak 's|instantsearch.js/es|instantsearch.js/cjs|g' {} + && find dist/cjs -name '*.bak' -type f -exec rm {} +",
```

Comment:
We could move this to our `script/`. It would be more self-documenting and centralized somewhere when edits are needed.